errai-ioc/src/test/java/org/jboss/errai/ioc/rebind/ioc/bootstrapper/IOCProcessorTest.java

```diff
@@ -0,0 +1,64 @@
+package org.jboss.errai.ioc.rebind.ioc.bootstrapper;
+
+import org.jboss.errai.ioc.client.api.EnabledByProperty;
+import org.jboss.errai.ioc.rebind.ioc.injector.api.InjectionContext;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+@RunWith(MockitoJUnitRunner.class)
+public class IOCProcessorTest {
```

Comment:
Since this currently only tests the property value, could you please rename it to something descriptive like `PropertyValueTest`?